deepchem/feat/vocabulary_builders/hf_vocab.py

```diff
@@ -0,0 +1,33 @@
+from deepchem.feat.vocabulary_builders.vocabulary_builder import VocabularyBuilder
+from tokenizers import Tokenizer
```

Comment:
Is huggingface already an optional dependency?

Reply:
yes